modules/ROOT/pages/dataweave-variables.adoc

```diff
@@ -141,35 +141,37 @@ The header of a DataWeave script accepts a `var` directive that initializes a va
 +
 * Local variables are initialized in the body of the DataWeave script and can be referenced by name only from within the scope of the expression where they are initialized.
 +
-The syntax for initializing a local variable looks like this:
-`using (<variable-name> = <expression>)`
-+
-It must use the keyword `using`, and it must use the `var` syntax, rather than the `fun` syntax.
+The syntax that MuleSoft recommends for initializing a local variable looks like this:
```

Comment:
This `do (<variable-name> = <expression>)` is not correct is 

```
do {
[type <name> = <type expression>]*
[var <name> = <expression>]*
[fun <name>(<params>*) = <expression>]*
[ns <name> <nsUri>]*
---
<body>
}
```

Reply:
done